lib/client/connection.js

```diff
@@ -116,11 +115,11 @@ Connection.prototype.bindToSocket = function(socket) {
       var data = (typeof event.data === 'string') ?
         JSON.parse(event.data) : event.data;
     } catch (err) {
-      console.warn('Failed to parse message', event);
+      logger.warn('Failed to parse message', event);
       return;
     }
 
-    if (connection.debug) console.log('RECV', JSON.stringify(data));
+    logger.debug('RECV', JSON.stringify(data));
```

Comment:
As we discussed, this is _technically_ a breaking change, so I'd be more comfortable leaving the `connection.debug` flag for now and putting this on the list of candidates for 1.0 breaking changes.

That list isn't actually written yet, but I'll make a note of it for when we formally compile one.